website/versioned_docs/version-29.5/TutorialjQuery.md

```diff
@@ -68,5 +68,18 @@ To get started with the JSDOM [test environment](Configuration.md#testenvironmen
 ```bash npm2yarn
 npm install --save-dev jest-environment-jsdom
 ```
+After installing, update your Jest configuration in `package.json`:
```

Comment:
should we just link to https://jestjs.io/docs/configuration/#testenvironment-string instead?

Reply:
I think it would be good to link to [the docs](https://jestjs.io/docs/configuration/#testenvironment-string) **as well**. I meant to add this as a quick note without having to go through the lengthy docs.
 I will defer to your judgment on this. I have no strong preferences.